### PR TITLE
[WIP] Add API method subscriptions.find_all(account_id)

### PR DESCRIPTION
**Pull request by nepalez**
CP has implemented new method to return all subscriptions http://cloudpayments.ru/Docs/Api#find-recurrent, so we can add it to the client.

### CODE REVIEW

**Comment by undr** [comment]
@nepalez Thank you for your contribution. 
Please, do rebase this PR with `master` in order to fix travis-ci issue. And I'll merge and release new version as soon as all tests will be green =)

**Comment by nepalez** [comment]
Actually, I have all tests failed on my local machine in exactly the same way travis failed for `ruby mri 2.3`

There are 6 months from the last release, so we can deal with some breaking changes in dependencies such as webmock. I need a bit more time to expore this

PS. Providing https://hacktoberfest.digitalocean.com/ could you add a "Hacktoberfest" label for this PR?

**Comment by undr** [comment]
@nepalez I've fixed issues related to `webmock`. Just do rebase.

File: spec/cloud_payments/namespaces/subscriptions_spec.rb [hunk 1]
```diff
@@ -21,6 +21,25 @@
     end
   end
 
+  describe '#find_all', :focus do
```
Comment (undr) [inline spec/cloud_payments/namespaces/subscriptions_spec.rb]: I suppose, now there is no need to use `:focus` tag here

Comment (nepalez) [reply spec/cloud_payments/namespaces/subscriptions_spec.rb]: OMG

**Comment by undr** [comment]
Awesome! Thanks.

**Pull request merged**